cli/command/bundlefile/bundlefile.go

```diff
@@ -14,15 +14,16 @@ type Bundlefile struct {
 
 // Service is a service from a bundlefile
 type Service struct {
-	Image      string
-	Command    []string          `json:",omitempty"`
-	Args       []string          `json:",omitempty"`
-	Env        []string          `json:",omitempty"`
-	Labels     map[string]string `json:",omitempty"`
-	Ports      []Port            `json:",omitempty"`
-	WorkingDir *string           `json:",omitempty"`
-	User       *string           `json:",omitempty"`
-	Networks   []string          `json:",omitempty"`
+	Image        string
+	Command      []string          `json:",omitempty"`
+	Args         []string          `json:",omitempty"`
+	Env          []string          `json:",omitempty"`
+	Labels       map[string]string `json:",omitempty"`
+	EndpointMode *string           `json:",omitempty"`
```

Comment:
Let's not add it to the bundlefile. The bundle is supposed to a subset of options, and I'm not sure it's appropriate.